cmd/frostfs-adm/internal/modules/morph/internal/types.pb.go

```diff
@@ -21,7 +21,7 @@ const (
 	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
 )
 
-// Client group identifier in the NeoFS subnet.
```

Comment:
Change the orig `.pb` file instead of the generated one?

Reply:
Fixed